src/test/java/org/openhab/binding/zwave/test/internal/protocol/commandclass/ZWaveClockCommandClassTest.java

```diff
@@ -75,4 +75,24 @@ public void setTime() {
         assertEquals(msg.getMessagePayload()[5], expectedResponse[5]);
     }
 
+    @Test
+    public void getReportMessage() {
+        ZWaveClockCommandClass cls = (ZWaveClockCommandClass) getCommandClass(CommandClass.CLOCK);
+
+        byte[] expectedResponse = { 99, 4, -127, 6, -128, 0 };
+
+        Calendar utc = Calendar.getInstance();
+        utc.setTimeZone(TimeZone.getTimeZone("UTC"));
+        utc.setTime(new Date(0));
+        SerialMessage msg = cls.getReportMessage(utc);
+
+        assertTrue(Arrays.equals(msg.getMessagePayload(), expectedResponse));
+
+        assertEquals(msg.getMessagePayload()[0], expectedResponse[0]);
```

Comment:
Why is there both the arrays.equals chck, and individual checks against the array. Surely the last 6 asserts aren't needed?